apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -391,6 +395,86 @@ schema("/clients/:clientid/keepalive") ->
             }
         }
     };
+schema("/clients/:clientid/mqueue_messages") ->
+    #{
+        'operationId' => mqueue_msgs,
+        get => #{
+            description => ?DESC(get_client_mqueue_msgs),
+            tags => ?TAGS,
+            parameters => [
+                {clientid, hoconsc:mk(binary(), #{in => path})},
+                {payload,
+                    hoconsc:mk(hoconsc:union([none, base64, plain]), #{
+                        in => query, default => base64
+                    })},
+                {max_payload_bytes,
+                    hoconsc:mk(emqx_schema:bytesize(), #{
+                        in => query, default => <<"1MB">>
+                    })},
+                hoconsc:ref(emqx_dashboard_swagger, continuation),
+                hoconsc:ref(emqx_dashboard_swagger, limit)
+            ],
+            responses => #{
+                200 =>
+                    emqx_dashboard_swagger:schema_with_example(?R_REF(messages), #{
+                        <<"data">> => [message_example()],
+                        <<"meta">> => #{
+                            <<"count">> => 1,
+                            <<"continuation">> => <<"AAYS53qRa0n07AAABFIACg==">>,
+                            <<"limit">> => 50,
+                            <<"hasnext">> => false
+                        }
+                    }),
+                400 =>
+                    emqx_dashboard_swagger:error_codes(
+                        ['INVALID_PARAMETER'], <<"Invalid parameters">>
+                    ),
+                404 => emqx_dashboard_swagger:error_codes(
+                    ['CLIENTID_NOT_FOUND'], <<"Client ID not found">>
+                )
+            }
+        }
+    };
+schema("/clients/:clientid/inflight_messages") ->
+    #{
+        'operationId' => inflight_msgs,
+        get => #{
+            description => ?DESC(get_client_inflight_msgs),
+            tags => ?TAGS,
+            parameters => [
+                {clientid, hoconsc:mk(binary(), #{in => path})},
+                {payload,
+                    hoconsc:mk(hoconsc:union([none, base64, plain]), #{
+                        in => query, default => base64
+                    })},
+                {max_payload_bytes,
+                    hoconsc:mk(emqx_schema:bytesize(), #{
+                        in => query, default => <<"1MB">>
+                    })},
+                hoconsc:ref(emqx_dashboard_swagger, continuation),
+                hoconsc:ref(emqx_dashboard_swagger, limit)
+            ],
+            responses => #{
+                200 =>
+                    emqx_dashboard_swagger:schema_with_example(?R_REF(messages), #{
+                        <<"data">> => [message_example()],
+                        <<"meta">> => #{
+                            <<"count">> => 1,
+                            <<"continuation">> => <<"AAYS53qRa0n07AAABFIACg==">>,
+                            <<"limit">> => 50,
```

Comment:
there is maybe no need to return `limit` in response ?

Reply:
Removed `limit` from meta